### DISCUSSION TITLE
Add a button to toggle to "Boot Up" sound

### DISCUSSION THREAD

**Discussion by John-Deligiannis** [seerge/g-helper#717] [Ideas]
I was thinking it might be useful to add a button to toggle the boot-up sound (I'm using a g14 2021). I think I remember this being an option in Armory Crate, and I think it might also be possible to change it in the bios. Nevertheless, it would be nice to not have to go into the bios to turn the sound off.

**Comment by seerge** [comment]
@John-Deligiannis hello, this is a duplicate of https://github.com/seerge/g-helper/issues/184
There is no endpoint that I would know that can control that :( (+ you can simply disable it in bios)

**Reply by John-Deligiannis** [reply]
Ok, thanks.

**Reply by seerge** [reply]
Added in https://github.com/seerge/g-helper/releases/tag/v0.120
